src/components/bookRenderer.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { api } from "~/utils/api";
+import { BellIcon, CheckIcon } from "@radix-ui/react-icons";
+
+import { cn } from "~/lib/utils";
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+type CardProps = React.ComponentProps<typeof Card>;
+
+const BookRenderer = ({ className, ...props }: CardProps) => {
```

Comment:
so this doesn't really need to take in `CardProps` right? Nor does it need `className`? What does the bookRenderer really need to take in? that should be the component's props.